package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "@fastify/cors": "^11.0.1",
+    "react-hook-form": "^7.59.0",
```

Comment:
maybe this dependency should be in frontend package.json? Would make sense since it is only used there.